test/system/src/test/java/io/pravega/test/system/PravegaTest.java

```diff
@@ -72,55 +69,42 @@
      */
     @Environment
     public static void initialize() {
+        log.info("Environment: Invoking tests with security enabled: {}", Utils.SECURITY_ENABLED);
         URI zkUri = startZookeeperInstance();
         startBookkeeperInstances(zkUri);
         URI controllerUri = ensureControllerRunning(zkUri);
         ensureSegmentStoreRunning(zkUri, controllerUri);
     }
 
-    @BeforeClass
-    public static void beforeClass() {
-        // This is the placeholder to perform any operation on the services before executing the system tests
-    }
-
     /**
-     * Invoke the createStream method, ensure we are able to create stream.
+     * Invoke the simpleTest, ensure we are able to produce  events.
+     * The test fails incase of exceptions while writing to the stream.
+     *
      */
-    @Before
-    public void createStream() {
+    @Test
+    public void simpleTest() {
+        log.info("simpleTest:: with security enabled: {}", Utils.SECURITY_ENABLED);
```

Comment:
Please avoid adding method names in log statements. Also, should this really be an info level log? Debug or trace level looks more appropriate to me here. 

Can we use a better name for the test - one that reveals the intent of the test?

Reply:
Will remove method name from log and change to debug.